docs/upload_mapping_file.md

```diff
@@ -31,7 +31,7 @@ plugins {
 ```bash
 ./gradlew uploadMappingRelease
 ```
-   **Note**: If your project uses additional flavors, the plugin will provide an upload task for each variant with obfuscation enabled.
+**Note**: If your project uses additional flavors, the plugin will provide an upload task for each variant with obfuscation enabled. In this case please also make sure to initialize SDK with a proper variant name (the necessary API is available since version `1.8.0`).
```

Comment:
```suggestion
**Note**: If your project uses additional flavors, the plugin provides an upload task for each variant with obfuscation enabled. In this case, initialize SDK with a proper variant name (the necessary API is available since version `1.8.0`).
```